plenum/test/consensus/helper.py

```diff
@@ -0,0 +1,70 @@
+from typing import Optional, List
+
+from plenum.common.config_util import getConfig
+from plenum.common.event_bus import InternalBus
+from plenum.server.consensus.replica_service import ReplicaService
+from plenum.server.database_manager import DatabaseManager
+from plenum.server.ledgers_bootstrap import LedgersBootstrap, IN_MEMORY_LOCATION
+from plenum.server.node import Node
+from plenum.server.request_managers.read_request_manager import ReadRequestManager
+from plenum.server.request_managers.write_request_manager import WriteRequestManager
+from plenum.test.greek import genNodeNames
+from plenum.test.helper import MockTimer
+from plenum.test.simulation.sim_network import SimNetwork
+from plenum.test.simulation.sim_random import SimRandom, DefaultSimRandom
+from plenum.test.testing_utils import FakeSomething
+
+
+class TestLedgersBootstrap(LedgersBootstrap):
+    def create_bls_bft(self):
+        return FakeSomething()
```

Comment:
Should it be implementation of BlsBft interface?

Reply:
Yes, this is planned after all other things start working :)